projects/HieraSeg_project/decode_head/sep_aspp_contrast_head.py

```diff
@@ -0,0 +1,214 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+from typing import List
+
+import torch
+import torch.nn as nn
+from mmcv.cnn import ConvModule, DepthwiseSeparableConvModule, build_norm_layer
+from torch import Tensor
+
+from mmseg.registry import MODELS
+from ..losses import accuracy
+from ..utils import resize
+from .aspp_head import ASPPHead, ASPPModule
```

Comment:
Might import from mmseg, not the relative path, since the `projects` folder doesn't contain these modules.